server/datastore/mysql/cron_stats.go

```diff
@@ -50,13 +50,13 @@ func (ds *Datastore) UpdateCronStats(ctx context.Context, id int, status fleet.C
 }
 
 func (ds *Datastore) CleanupCronStats(ctx context.Context) error {
```

Comment:
@roperzh, I've noticed that the test for this sometimes [flakes](https://github.com/fleetdm/fleet/actions/runs/3490675756/jobs/5842396181#step:9:4016), but it seems to be limited to the MySQL 8 version of the test and I don't see it on the MySQL 5.7 version. All I can think is maybe there is a difference in how they handle date comparisons and/or difference in our db setup for the different tests. Do you have any ideas?

Reply:
@gillespi314 interesting! looking at the documentation of both MySQL versions I don't see anything that would indicate that.

My gut feeling is that the cron stats were coming in a different order than the one expected by the test. I see you added an `ORDER BY id` below, I'm curious: was the test still failing even with that?

Reply:
@roperzh, I haven't seen any more failures for that test since adding the `ORDER BY id` 🤞

Reply:
nice! in that case, should we revert the `NOW()` / `UTC_TIMESTAMP()` change? just to be safe we're not changing anything besides the tests. I'm also OK merging it as-is.

Reply:
Good idea! I'll revert and we'll see if the test still passes.